src/core/server/base.js

```diff
@@ -333,27 +339,29 @@ function baseServer(routes, createStore, { appSagas, appInstanceName = appName }
         because a response was already sent; error: ${error}`);
       return next(error);
     }
+
     log.error(`Showing 500 page for error: ${error}`);
     log.error({ err: error }); // log the stack trace too.
-    return showErrorPage({ createStore, error, status: 500, req, res });
+    return showErrorPage({ createStore, error, status: 500, req, res, config });
   });
 
   return app;
 }
 
 export function runServer({
-  listen = true, app = appName, exitProcess = true,
```

Comment:
Hmm, I can't see where any callers were using this `app` parameter. I guess it's safe to remove.